compiler/tests/fail-functag.sp

```diff
@@ -0,0 +1 @@
+functag public:Action(Handle:handle);
```

Comment:
The syntax looks wrong. Not that it really matters since it already fails on `functag`..
Same for the funcenum one.